pkg/auth/fake/auth.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright (c) 2018 Bitnami
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fake
+
+import (
+	authUtils "github.com/kubeapps/kubeapps/pkg/auth"
+)
+
+type FakeAuth struct {
+	ForbiddenActions []authUtils.Action
+}
+
+func (f *FakeAuth) Validate() error {
```

Comment:
Have you considered using https://github.com/stretchr/testify/mock in these fake implementations? Though not really that important for these classes, it might help make using this a bit more flexible and we can also do things like assert the number of times a method is called, or assert that certain arguments were passed to a method. See https://github.com/kubeapps/common/blob/master/datastore/mockstore/mockstore.go and https://github.com/kubeapps/kubeapps/blob/master/cmd/chart-repo/utils_test.go#L282 for an example of how this is used.

Reply:
sounds good 👍